row_061_app.py

```
@@ -0,0 +1,46 @@
+        f.save(os.path.join(app.config['UPLOADED_PATH'], f.filename))
+    return render_template('index.html')
+
+
+@app.route('/completed')
+def completed():
+# {/fact}
+    return '<h1>The Redirected Page</h1><p>Upload completed.</p>'
+
+
+if __name__ == '__main__':
+    app.run(debug=True)
```

Ruff (0.13.2)

Use of `debug=True` in Flask app detected

(S201)